lib/Field/stories/FieldStory.js

```diff
@@ -36,65 +41,143 @@ storiesOf('Components', module).add('Field', () => {
     },
     'ltr'
   );
-  const canEdit = boolean('Can edit', false);
+  const inStructureEditMode = boolean('In Structure Edit Mode', false);
 
+  const label = text('Field label', 'Field label text');
+  const isActive = boolean('Is active', true);
+  const instructions = text(
+    'Instructions',
+    "Instruction text to help inform the user of what they're meant to write in this field. http://gathercontent.com"
+  );
   return (
-    <StoryItem
-      title="Field"
-      description="A field component provides supporting UI for gathering content. The child component should be responsible for handling the gathering experience but this component renders labels, actions and validations to support that experience."
-    >
-      <Field
-        fieldId="123"
-        actions={mockActions}
-        validations={mockValidations}
-        label={text('Field label', 'Field label text')}
-        instructions={text(
-          'Instructions',
-          "Instruction text to help inform the user of what they're meant to write in this field. http://gathercontent.com"
-        )}
-        hasFormatting={boolean('Has formatting', true)}
-        isActive={boolean('Is active', true)}
-        disabled={boolean('Disabled', false)}
-        canEdit={canEdit}
-        dir={dir}
+    <>
+      <StoryItem
+        title="FieldNew"
+        description="A newer, Field component, can contain Content and Meta"
+      >
+        <FieldNew
+          className={cx({ 'field-active': isActive })}
+          dir={dir}
+          inStructureEditMode={inStructureEditMode}
+        >
+          <FieldNew.Meta className="">
+            <FieldLabel
```

Comment:
🚩 What do you think to doing `<FieldNew.Label />` here? As storybook is the only source of documentation, I think it would be great to push people towards using this method!